public/src/main/java/com/revenuecat/purchases/models/StoreProduct.kt

```diff
@@ -1,40 +1,36 @@
 package com.revenuecat.purchases.models
 
 import android.os.Parcelable
+import com.android.billingclient.api.SkuDetails
 import com.revenuecat.purchases.ProductType
-import com.revenuecat.purchases.parceler.JSONObjectParceler
-import kotlinx.parcelize.Parcelize
-import kotlinx.parcelize.TypeParceler
 import org.json.JSONObject
 
 /**
  * Represents an in-app product's or subscription's listing details.
  */
-@Parcelize
-@TypeParceler<JSONObject, JSONObjectParceler>()
-data class StoreProduct(
+interface StoreProduct : Parcelable {
     /**
-     * The product ID.
+     * sku for BC4 and Amazon, subId for BC5
```

Comment:
Since this would be public documentation we should improve this. Also, I think even on BC4, this would be the subscription ID in the google dashboard. Maybe:
```suggestion
     * sku for Amazon Store, subscriptionId for Play Store
```